crawlerr/services/rag_service.py

```
@@ -341,6 +586,86 @@ async def _rerank_results(
         if not matches:
             return matches
         
+        try:
+            if self.reranker_type == "qwen3":
+                return await self._rerank_with_qwen3(query, matches, top_k)
+            elif self.reranker_type == "custom":
```

Ruff (0.12.2)

Unnecessary `elif` after `return` statement

Remove unnecessary `elif`

(RET505)